### PR TITLE
Support Bloop pipelining

### PR DESCRIPTION
**Pull request by propensive**

### CODE REVIEW

**Comment by propensive** [comment]
Pipelining causes multi-module builds to produce output in a different order. It's probably still stable, though.

**Comment by odisseus** [comment]
Looks good, but the integration tests that have multiple modules are now broken.

File: src/core/args.scala [hunk 1]
```diff
@@ -74,6 +74,7 @@ object Args {
   val LicenseArg = CliParam[LicenseId]('L', 'license, "license for code in this project")
   val ModuleArg = CliParam[ModuleId]('m', 'module, "specify a module")
   val MainArg = CliParam[String]('M', 'main, "specify a main class")
+  val PipeliningArg = CliParam[Boolean]('p', Symbol("pipelining"), "use compilation pipelining (on, off) (experimental)")
```
Comment (odisseus) [inline src/core/args.scala]: This argument has the same letter as `ProjectArg`, and some commands accept both of them.

**Comment by odisseus** [comment]
While rebuilding this pull request with itself, I got the following error:
```
[E] fury/schema>local:src/schema/schema.scala +91:
  | not found: value 
  | rows      <- ~Diff.gen[Schema].diff(schema, other)
```
When it happened, the compilation of `fury/core` was not completed yet.

Rebuilding the same code with make gave no errors.

**Comment by propensive** [comment]
I think that the failure is acceptable. It's probably a bug in Bloop, and it fits with the decision to mark pipelining as "experimental".

**Pull request merged**